### DISCUSSION TITLE
Distractor-aware incremental learning

### DISCUSSION THREAD

**Discussion by maxizi** [STVIR/pysot#324] [General] [answered]
Hello together,

I am currently investigating the DaSiamRPN architecture, which I thought is implemented in the scope of pysot. I can find the long-term module which implements the long-term strategy described in the official paper and also the distractor-aware training.
However, I cannot find the distractor-aware incremental learning part (section 3.3 in the paper). Did I maybe overread it somewhere or is it not implemented?

Thanks for your help!

**Comment by iminfine** [comment, marked as answer]
It seems there is not an official implement. There is an unofficial implement at:https://github.com/JachinShen/DaSiamRPNAT
